lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -147,9 +151,22 @@ export async function doAutoReplace(
   if (reuseExistingBranch) {
     return await checkExistingBranch(upgrade, existingContent);
   }
+  const replaceWithoutReplaceString = Boolean(
+    newName &&
+      newName !== depName &&
+      (!upgrade.replaceString ||
+        upgrade.replaceString?.indexOf(depName!) === -1)
```

Comment:
This is a little dense, and hard to parse. Would you be open to using `@sindresorhus/is` to clarify the meaning a little? For example:

- `is.string(newName)` instead of `newName`
- `is.undefined(upgrade.replaceString)` instead of `!upgrade.replaceString`
- `upgrade.replaceString?.includes(depName)` instead of `upgrade.replaceString?.indexOf(depName!) === -1`